test/rendering/cases/10-geojson/spec.json

```diff
@@ -3,6 +3,21 @@
     {
       "type": "GeoJSON",
       "attribution": "",
+      "style": {
+        "name": "Test Style",
+        "rules": [
+          {
+            "name": "Rule 1",
+            "symbolizers": [
+              {
+                "kind": "Line",
+                "color": "black",
```

Comment:
@jahow FYI I add this error with unit test when using `"#000000"` instead of `"black"`.

```
uncaught exception [Error: SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at eval (webpack: ///./test/rendering/testbench/print.js?:6:19)
    at Module../test/rendering/testbench/print.js (http: //localhost:8888/print.js:9148:1)
    at __webpack_require__ (http: //localhost:8888/print.js:20:30)
    at eval (webpack: ///multi_(webpack)-dev-server/client?:2:18)
    at Object.0 (http: //localhost:8888/print.js:9159:1)
    at __webpack_require__ (http: //localhost:8888/print.js:20:30)
    at http: //localhost:8888/print.js:84:18
    at http: //localhost:8888/print.js:87:10]
console.log: [inkmap
    ] Ready, using worker

```

Reply:
Strange, I'll look into it